sklearn/utils/estimator_checks.py

```diff
@@ -204,7 +205,7 @@ def set_fast_parameters(estimator):
             and estimator.__class__.__name__ != "TSNE"):
         estimator.set_params(n_iter=5)
     if "max_iter" in params:
-        # NMF
+        warnings.simplefilter("ignore", ConvergenceWarning)
```

Comment:
Because "max_iter" is now in the params of SAG, this no longer applies exclusively to NMF. I suppose it would have been wiser to instead make the comment "NMF or SAG".